ui_src/src/domain/administration/versionUpgrade/index.js

```diff
@@ -82,6 +102,15 @@ function VersionUpgrade() {
                     <div className="banner-section">
                         <img src={upgradeBanner} width="97%" alt="upgradeBanner" />
                         <div className="actions">
+                            <div className="current-version-wrapper">
+                                <version
+                                    is="x3d"
+                                    style={{ cursor: !state.isLatest ? 'pointer' : 'default' }}
+                                    onClick={() => (!state.isLatest ? history.push(`${pathDomains.administration}/version_upgrade`) : null)}
```

Comment:
The onClick method will direct you to the same page. We don't need the 'Current version' will be clickable